portfolio/src/main/webapp/script.js

```diff
@@ -29,7 +29,12 @@ function addRandomFact() {
 }
 
 function FetchDispGreeting (){
-    fetch("/data").then(response => response.text()).then((greeting) => {
-        document.getElementById("greeting").innerHTML = greeting;
+      fetch("/data").then(response => response.json()).then((messages) => {
+            var Affirmations = document.getElementById("messages");
```

Comment:
nit: generally for variable names we start them with lowercase. Usually only class names are started with uppercase.

Reply:
Done! Thanks for the tip